pkg/monitoring/rules/alerts/alerts.go

```diff
@@ -0,0 +1,57 @@
+package alerts
```

Comment:
@sradco mentioned this file can be named prometheus.go

Reply:
done